commit bd14f37e6d0daa8f495c8ce998c03bfa012ff0ee
Author: Aurelien Jarno <aurelien@aurel32.net>
Date:   Thu Jan 4 14:40:11 2024 +0100

    Add lsusb.py.1 to DISTCLEANFILES
    
    Commit 83690ec408b2 ("Add a manpage for lsusb.py") added the lsusb.py.1
    manpage but forgot to add it to DISTCLEANFILES. Fix that.
    
    Signed-off-by: Aurelien Jarno <aurelien@aurel32.net>

diff --git a/Makefile.am b/Makefile.am
index d3a9025..02c7a33 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -82,6 +82,7 @@ usb-devices.1: $(srcdir)/usb-devices.1.in
 
 DISTCLEANFILES = \
 	lsusb.py \
+	lsusb.py.1 \
 	lsusb.8 \
 	usb-devices.1 \
 	usbhid-dump.8
